### PR TITLE
fix(plan): update scanning to cover more areas

### PR DESCRIPTION
**Pull request by owenrumney**
Signed-off-by: Owen Rumney <owen.rumney@aquasec.com>

### CODE REVIEW

File: pkg/scanners/terraformplan/scanner.go [hunk 1]
```diff
@@ -54,6 +55,12 @@ func (s *Scanner) Scan(reader io.Reader) (scan.Results, error) {
 		return nil, err
 	}
 
-	scanner := terraformScanner.New()
+	content, err := planFS.ReadFile("main.tf")
+	if err != nil {
+		panic(err)
```
Comment (liamg) [inline pkg/scanners/terraformplan/scanner.go:60]:
```suggestion
		return nil, err
```

File: pkg/scanners/terraformplan/scanner.go [hunk 2]
```diff
@@ -54,6 +55,12 @@ func (s *Scanner) Scan(reader io.Reader) (scan.Results, error) {
...
+	}
+	fmt.Printf("Content: %s\n", string(content))
```
Comment (liamg) [inline pkg/scanners/terraformplan/scanner.go:62]: I think this is debug?

**Review by liamg — CHANGES_REQUESTED** [review]
Couple of small comments, otherwise AOK

File: pkg/scanners/terraformplan/scanner.go (hunk 2 above)
Comment (owenrumney) [reply pkg/scanners/terraformplan/scanner.go]: yeah, my bad. Removed

**Review by liamg — APPROVED** [review]

**Pull request merged**